starter-code/js/index.js

```diff
@@ -1,52 +1,124 @@
+var productName = document.querySelector("#productName>span");
 
-}
+var unitPrice = document.querySelector(".unitPrice>span");
 
-function getPriceByProduct(itemNode){
+var vraperKing = document.querySelector(".wrapperKing");
 
-}
+var eachRow = document.querySelectorAll(".wrapper");
 
-function updatePriceByProduct(productPrice, index){
+var totalPrice = document.querySelector("h2>span");
 
-}
+var calcPricesButton = document.querySelector("#calcPrices>.btn");
 
-function getTotalPrice() {
+var input = document.querySelector("input");
 
-}
+var inputNewName = document.querySelector(".productName input");
 
-function createQuantityInput(){
+var btnCreate = document.querySelector(".btn-create");
 
-}
+var addClassName = function(element, name) {
+  element.className = name;
+};
 
-function createDeleteButton(){
 
-}
+function calcPriceForEachRow(rows) {
+  var totalPriceValue = 0;
 
-function createQuantityNode(){
+  rows.forEach(function(row) {
+    var productPriceNumber = 0;
+    var totalPriceRow = row.querySelector(".totalPrice>span");
+    var unitPrice = row.querySelector(".unitPrice span");
+    var totalPricePerItem = row.querySelector(".totalPrice>span");
+    var inputNumber = Number(row.querySelector("input").value);
 
-}
+    productPriceNumber += Number(unitPrice.textContent);
 
-function createItemNode(dataType, itemData){
+    function getPriceByProduct(element) {
+      totalPricePerItem.textContent = `${inputNumber * element}.00`;
+    }
+    
+    getPriceByProduct(productPriceNumber);
+
+    totalPriceValue += Number(totalPriceRow.textContent);
+  });
+  totalPrice.textContent = `${totalPriceValue}.00`;
+}
 
+function totalPriceValue(rows) {
+  rows.forEach(function(row) {
+    var totalPrice = 0;
+    var totalPriceRow = row.querySelector(".totalPrice>span");
+    totalPrice += Number(totalPriceRow.textContent);
+  });
 }
 
-function createNewItemRow(itemName, itemUnitPrice){
+function createRow() {
+  var newDiv = document.createElement("div");
 
+  var inputNewPrice = document.querySelector(".unitPrice input");
+
+  function createDiv(element) {
+    newDiv;
+    element.appendChild(newDiv);
+    return newDiv;
+  }
+
+  function createRowBody() {
+    var wrapper = createDiv(wrapperKing);
+    addClassName(wrapper, "wrapper");
+    wrapper.innerHTML = `
+  
+  <div class="productName">
+    <span>${inputNewName.value}</span>
+  </div>
+  
+  <div class="unitPrice">
+    $<span>${inputNewPrice.value}</span>
+  </div>
+
+  <div class="quantity">
+    <label for="">QTY</label>
+    <input type="number" name="" value="1">
+  </div>
+
+    <div class="totalPrice">
+      $<span>0.00</span>
+    </div>
+
+    <div>
+    <button class="btn btn-delete">
+      Delete
+    </button>
+  </div>
+  `;
+  }
+  createRowBody();
+};
+
+function selectDeleteButton(eachrow) {
+  eachrow.forEach(function(row) {
+    var deleteBtn = row.querySelector(".btn-delete");
+    function deleteRow() {
+      row.remove(row);
+    }
```

Comment:
It could be a nice addition if you add your 
`calcPriceForEachRowResponsive();`
function just after you remove the row so you don't have to click on the calculate price button again :).